spec/indexers/generic_work_indexer/date_values_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+RSpec.describe GenericWorkIndexer::DateValues do
+  # Tried using 'build' instead of 'create', sadly didn't work, dates
+  # get thrown out. This does slow it down a lot cause Fedora. :(
```

Comment:
I guess it is, I got it working with 'build'.  dates still get persisted though. Should prob elmiminate that comment, yeah.